Java script/storage assignment/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Local</h1>
+    <p id ="local" >Count</p>
+    <input type="button" name="local" id="" value="add to local" onclick="local_click()">
+    <h1>Session</h1>
+    <p id ="session">Count</p>
+    <input type="button" name ="session" value ="add to session" onclick="session_click()">
+</body>
+<script>
+let local = document.getElementById("local")
```

Comment:
Make sure to create a seperate js file instead of writing in script tag..

Reply:
As of now i approved but from next PR make a seperate js file

Reply:
Okay will create a new js file from next time 
Thank you